content/ja/handbook/company/working-groups/cross-functional-prioritization.md

```
@@ -0,0 +1,111 @@
+## 属性
+
+| プロパティ        | 値      |
+|-----------------|------------|
+| 作成日           | 2022-04-13 |
+| 目標終了日        | 2023-01-31 |
+| Slack           | [#wg_cross-functional-prioritization](https://gitlab.slack.com/archives/C03AWM7780G) |
+| Google Doc      | [ワーキンググループ アジェンダ](https://docs.google.com/document/d/1wog8bR7jg6SECefx9BGqIa07sFm_sXJPvelVAganYmc/edit#heading=h.pmtw3ocv2aty)  |
+| Issue ボード     | [Issue ボード](https://gitlab.com/groups/gitlab-com/-/boards/4199535?not[label_name][]=wg-cross-functional-prioritization-adoption&label_name[]=wg-crossfunctional-prioritization) |
+| ワーキンググループ用オープン MR | [MR リスト](https://gitlab.com/gitlab-com/www-gitlab-com/-/merge_requests?label_name%5B%5D=wg-cross-functional-prioritization) |
+| OKR | クロスファンクショナル優先順位付けモデルの成熟化 |
+| 概要 & ステータス | 下記の[終了条件](#exit-criteria)を参照 |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)